target_chains/solana/pyth_solana_receiver_sdk/Cargo.toml

```diff
@@ -13,8 +13,11 @@ name = "pyth_solana_receiver_sdk"
 
 
 [dependencies]
-anchor-lang = "0.31.1"
+anchor-lang = "0.32.1"
 hex = "0.4.3"
 pythnet-sdk = { path = "../../../pythnet/pythnet_sdk", version = "2.3.1", features = [
     "solana-program",
 ] }
+
+[dev-dependencies]
+solana-program = "2.3.0"
```

Comment:
maybe use `solana-borsh` for smaller footprint `solana_program::borsh0_10` is a reexport of `solana_borsh::v0_10`